Userland/Libraries/LibProtocol/RequestClient.h

```diff
@@ -24,6 +24,7 @@ class RequestClient final
 
 public:
     explicit RequestClient(NonnullOwnPtr<Core::LocalSocket>);
+    ~RequestClient();
```

Comment:
I think this one should also be `virtual override`? By way of `RequestClientEndpoint::Stub`